ush/config_defaults.yaml

```diff
@@ -1911,44 +1989,51 @@ task_run_prdgen:
   # OMP_STACKSIZE_RUN_PRDGEN: 
   # Controls the size of the stack for threads created by the OpenMP implementation.
   #-----------------------------------------------------------------------
-  KMP_AFFINITY_RUN_PRDGEN: "scatter"
-  OMP_NUM_THREADS_RUN_PRDGEN: 1
-  OMP_STACKSIZE_RUN_PRDGEN: "1024m"
-  #-----------------------------------------------------------------------
-  # DO_PARALLEL_PRDGEN:
-  # Flag that determines whether to use CFP to run the product generation
-  # job in parallel. This should be used with the RRFS_NA_3km grid.
-  #
-  #-----------------------------------------------------------------------
-  DO_PARALLEL_PRDGEN: false
-  #
-  #
-  #-----------------------------------------------------------------------
-  #
-  # Set additional output grids for wgrib2 remapping, if any.
-  # Space-separated list of strings, e.g., ( "130" "242" "clue" )
-  # Default is no additional grids
-  #
-  # Current options as of 23 Apr 2021:
-  #  "130"   (CONUS 13.5 km)
-  #  "200"   (Puerto Rico 16 km)
-  #  "221"   (North America 32 km)
-  #  "242"   (Alaska 11.25 km)
-  #  "243"   (Pacific 0.4-deg)
-  #  "clue"  (NSSL/SPC 3-km CLUE grid for 2020/2021)
-  #  "hrrr"  (HRRR 3-km CONUS grid)
-  #  "hrrre" (HRRRE 3-km CONUS grid)
-  #  "rrfsak" (RRFS 3-km Alaska grid)
-  #  "hrrrak" (HRRR 3-km Alaska grid)
-  #
-  #-----------------------------------------------------------------------
-  #
-  ADDNL_OUTPUT_GRIDS: []
+  envvars:
+    KMP_AFFINITY_RUN_PRDGEN: "scatter"
+    OMP_NUM_THREADS_RUN_PRDGEN: 1
+    OMP_STACKSIZE_RUN_PRDGEN: "1024m"
+    #-----------------------------------------------------------------------
+    # DO_PARALLEL_PRDGEN:
+    # Flag that determines whether to use CFP to run the product generation
+    # job in parallel. This should be used with the RRFS_NA_3km grid.
+    #
+    #-----------------------------------------------------------------------
+    DO_PARALLEL_PRDGEN: false
+    #
+    #
+    #-----------------------------------------------------------------------
+    #
+    # Set additional output grids for wgrib2 remapping, if any.
+    # Space-separated list of strings, e.g., ( "130" "242" "clue" )
+    # Default is no additional grids
+    #
+    # Current options as of 23 Apr 2021:
+    #  "130"   (CONUS 13.5 km)
+    #  "200"   (Puerto Rico 16 km)
+    #  "221"   (North America 32 km)
+    #  "242"   (Alaska 11.25 km)
+    #  "243"   (Pacific 0.4-deg)
+    #  "clue"  (NSSL/SPC 3-km CLUE grid for 2020/2021)
+    #  "hrrr"  (HRRR 3-km CONUS grid)
+    #  "hrrre" (HRRRE 3-km CONUS grid)
+    #  "rrfsak" (RRFS 3-km Alaska grid)
+    #  "hrrrak" (HRRR 3-km Alaska grid)
+    #
+    #-----------------------------------------------------------------------
+    #
+    ADDNL_OUTPUT_GRIDS: []
 
 #----------------------------
 # PLOT_ALLVARS config parameters
 #-----------------------------
 task_plot_allvars:
+  execution:
+    batchargs:
+      nodes: 1
+      tasks_per_node: 1
+      exclusive: True
```

Comment:
Why was `exclusive` added to the `config_defaults.yaml` file for `task_plot_allvars`?  I don't see this requirement for the `plot_allvars` tasks in the current develop.  While it's inclusion doesn't adversely affect experiments, on machines where `exclusive` isn't supported, it leaves warning messages.